### PR TITLE
Issue 2749, added alt text to cart icon so that it will say Items if …

### LINKED ISSUE #2749: [Templates] [Cosmetics] Add Alt text to cart
Add Alt text to 
`<svg data-src="{% static "images/cart.svg" %}" width="35" height="30"/>`
in **base.html**
details: https://www.w3.org/wiki/HTML/Usage/TextAlternatives

### PR DESCRIPTION
**Pull request by MartinSeibert**
I want to merge this change because...

Issue 2749 suggests that there should be a text alternative in case the cart icon doesn't load. I added a text default to "Items" in case the icon doesn't load.

Fixes #2749

### Pull Request Checklist

1. [ ] Privileged views and APIs are guarded by proper permission checks.
1. [ ] All visible strings are translated with proper context.
1. [ ] All data-formatting is locale-aware (dates, numbers, and so on).
1. [ ] Database queries are optimized and the number of queries is constant.
1. [ ] The changes are tested.
1. [ ] The code is documented (docstrings, project documentation).
1. [ ] GraphQL schema and type definitions are up to date.

### CODE REVIEW

File: templates/base.html [hunk 1]
```diff
@@ -149,7 +149,7 @@
                   {% trans "Your Cart" context "Main navigation item" %}
                 </span>
                 <div class="navbar__brand__cart__icon">
-                  <svg data-src="{% static "images/cart.svg" %}" width="35" height="30"/>
+                  <svg data-src="{% static "images/cart.svg" %}" width="35" height="30" alt="Items"/>
```
Comment (maarcingebala) [inline templates/base.html]: I think that according to the good practices of writing alt texts we should be more descriptive here and say rather something like "An icon of a shopping cart." or even "An icon of a shopping cart possibly with the number of items added to the cart".

Comment (patrys) [reply templates/base.html]: I think in this particular case the icon is purely decorative as the textual label besides it does a good job describing what the button does. Decorative elements should have an empty alt.

Comment (NyanKiyoshi) [reply templates/base.html]: Plus, the label right next to the icon is saying "Your cart" then we have the cart's item count badge; it's already really descriptive and comprehensible without any alt text, even for blind users' accessibility.

<p align='center'>screenshot</p>

Comment (NyanKiyoshi) [reply templates/base.html]: I'm not sure what's the question? You can edit the file and then commit. If you want to clean the commit history, you can rewrite the history using `git reset HEAD~1 --soft` (I would only recommend if you are used to `git` though), then commit over it and force push.

**Comment by maarcingebala** [comment]
In the review, it was suggested that alt text is not necessary in this case.

**Pull request closed without merging**